packages/tizen_bundle/lib/tizen_bundle.dart

```diff
@@ -0,0 +1,395 @@
+// Copyright 2022 Samsung Electronics Co., Ltd. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'dart:ffi';
+import 'dart:typed_data';
+
+import 'package:ffi/ffi.dart';
+import 'package:flutter/services.dart';
+import 'package:tizen_interop/4.0/tizen.dart';
+import 'package:tizen_log/tizen_log.dart';
+
+/// Bundle is a string based Dictionary ADT.
+/// A dictionary is an ordered or unordered list of key element pairs,
+/// where keys are used to locate elements in the list.
+class Bundle {
```

Comment:
I just found that this class can be implemented by simply extending the [MapMixin](https://api.flutter.dev/flutter/dart-collection/MapMixin-class.html) class. Please read its documentation for details. All you need to implement is its five members but you can still provide a custom implementation of, for example, `length` which is based on `bundle_get_count`.

Reply:
Done. I've changed the bundle implementation using MapMixin.
I think it's better than providing a custom implementation.
Thanks for all comments. It's vefy helpful for me.